internal/service/medialive/channel_encoder_settings_schema.go

```diff
@@ -1315,9 +1315,10 @@ func channelEncoderSettingsSchema() *schema.Schema {
 					},
 				},
 				"video_descriptions": {
-					Type:     schema.TypeSet,
+					Type:     schema.TypeList,
 					Optional: true,
 					Computed: true,
+					MaxItems: 1,
```

Comment:
`VideoDescriptions` in the API docs and on the SDK both indicate that this can be a list with more than one item.

https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/medialive@v1.30.2/types#EncoderSettings